code/src/Components/AlbumCover.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+import Radium from 'radium'
+import styles from '../styles'
+import AlbumCoverBackground from './AlbumCoverBackground'
+import AlbumCoverIcons from './AlbumCoverIcons'
+
+const AlbumCover = (props) => {
+  const [hoverBackgroundVisible, setHoverBackgroundVisible] = useState(false)
+  const [underline, setUnderline] = useState(false)
+  return (
+    <div>
+      <div style={styles.imgWrapper} onMouseEnter={() => setHoverBackgroundVisible(true)} onMouseLeave={() => setHoverBackgroundVisible(false)}>
+        <a href={props.linkAlbum}>
+          <img style={styles.img} src={props.image} alt="Album cover"></img>
+          {hoverBackgroundVisible && <AlbumCoverBackground />}
+          {hoverBackgroundVisible && <AlbumCoverIcons />}
+        </a>
+      </div>
+
+      <a href={props.linkAlbum} style={styles.linkText}>
+        <h2 key={"Album"} style={styles.h2}>{props.name}</h2>
+      </a>
+      <a href={props.linkArtist} style={styles.linkText}>
+        <h3 key={"Artist"} style={styles.h3}>{props.artist}</h3>
+      </a>
```

Comment:
Here, to be able to show multiple artists (if there are multiple artists on the album) do a similar solution as you did in App.js to get all the albums. 
```
{props.artists.map(artist => (
    <a className="artist" href={artist.external_urls.spotify}>{artist.name}</a>
))}
```

Reply:
Thank you for feedback! I will fix this.